Java/Greedy_Algorithms/FractionalKnapsack.java

```diff
@@ -0,0 +1,127 @@
+// You are given two lists of integers weights and values which have 
+// the same length and an integer capacity. weights[i] and values[i] 
+// represent the weight and value of the ith item. Given that you can 
+// take at most capacity weights, and that you can take a fraction of 
+// an item's weight with proportionate value, return the maximum amount
+// of value you can get, rounded down to the nearest integer.
+
+// Input
+
+// weights = [5, 6, 2]
+// values = [100, 100, 1]
+// capacity = 8
+
+// Output
+
+// 150
+
+// Explanation
+
+// Take the item with weight 5 and value 100, we are left with capacity 3
+// Take half of the item with weight 6, so it's value would become 50.
+// Thereby getting a maximum profit of 150
```

Comment:
Few changes: 
- Use multi-line comments 
- Shift sample I/O to the end of the code
- Add complexity at the end of the Code